tracer/contrib/sqltraced/sqltest/sqltest.go

```diff
@@ -12,16 +12,37 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// setupTestCase initializes MySQL or Postgres databases and returns a
+// teardown function that must be executed via `defer`
+func setupTestCase(t *testing.T, db *DB) func(t *testing.T, db *DB) {
+	// creates the database
+	db.Exec("DROP TABLE IF EXISTS city")
+	db.Exec("CREATE TABLE city (id integer NOT NULL DEFAULT '0', name text)")
```

Comment:
Is compatible both for MySQL and Postgres. No need to add extra fields since we should just check that a query is traced.